### ISSUE TITLE
FAIL: //tensorflow/python/kernel_tests:sparse_tensor_dense_matmul_grad_test

### ISSUE THREAD

**Issue by lukeiwanski** [lukeiwanski/tensorflow#46]
## System Info

```
  Name:						 Hawaii
  Vendor:					 Advanced Micro Devices, Inc.
  Device OpenCL C version:			 OpenCL C 2.0 
  Driver version:				 1912.5 (VM)
  Profile:					 FULL_PROFILE
  Version:					 OpenCL 2.0 AMD-APP (1912.5)
  Extensions:					 cl_khr_fp64 cl_amd_fp64 cl_khr_global_int32_base_atomics cl_khr_global_int32_extended_atomics cl_khr_local_int32_base_atomics cl_khr_local_int32_extended_atomics cl_khr_int64_base_atomics cl_khr_int64_extended_atomics cl_khr_3d_image_writes cl_khr_byte_addressable_store cl_khr_gl_sharing cl_khr_gl_depth_images cl_ext_atomic_counters_32 cl_amd_device_attribute_query cl_amd_vec3 cl_amd_printf cl_amd_media_ops cl_amd_media_ops2 cl_amd_popcnt cl_khr_image2d_from_buffer cl_khr_spir cl_khr_subgroups cl_khr_gl_event cl_khr_depth_images cl_khr_mipmap_image cl_khr_mipmap_image_writes

```
ComputeCpp 0.2.0

## To reproduce
```
bazel test --config=sycl --test_tag_filters=local,-benchmark-test --local_test_jobs=4 -k --test_lang_filters=cc,py --action_env=LD_PRELOAD=/usr/lib/libOpenCL.so.1 --test_timeout 300,750,1200,3600 //tensorflow/python/kernel_tests:sparse_tensor_dense_matmul_grad_test
```

## Error
```
FAIL: testGradients (__main__.SparseTensorDenseMatMulGradientTest)
...
low/python/kernel_tests/sparse_tensor_dense_matmul_grad_test.py", line 82, in _testGradients
    self.assertLess(err, 1e-3)
AssertionError: nan not less than 0.001


```

**Comment by jwlawson** [comment]
Fixed by #53

**Comment by lukeiwanski** [comment]
Awesome! Closing!
